tests/integration/integration_python_test.go

```diff
@@ -1329,3 +1329,32 @@ func TestFailsOnImplicitDependencyCyclesPython(t *testing.T) {
 	require.NoError(t, pt.TestLifeCycleInitialize(), "initialize")
 	require.Error(t, pt.TestPreviewUpdateAndEdits(), "preview")
 }
+
+// Test a paramaterized provider with python.
+//
+//nolint:paralleltest // ProgramTest calls t.Parallel()
+func TestParamaterizedPython(t *testing.T) {
+	e := ptesting.NewEnvironment(t)
+
+	// We can't use ImportDirectory here because we need to run this in the right directory such that the relative paths
+	// work.
```

Comment:
That's unfortunate. Could we import the test provider as well instead of doing that?  I guess it also needs the sdk?

Reply:
Well we need the integration test to pick up the SDK we write out as well and that makes its own environment, so this seems the simplest way to run this test. I'm hoping to turn these tests to conformance tests at some point so I wouldn't worry about the tidiness of this too much.